UIComponents/Package.swift

```diff
@@ -0,0 +1,28 @@
+// swift-tools-version: 5.6
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "UIComponents",
```

Comment:
You have decided to choose "UI Components" instead of "AnimealUIKit". What's the reason? I thought that we should put in such package both of UI components and styles.

Reply:
I see your point @diantyn , let's schedule a meeting tomorrow and discuss possible names/entities related to UI